src/controllers/scripting/legacy/scriptconnection.h

```diff
@@ -33,4 +33,6 @@ class ScriptConnection {
     inline bool operator!=(const ScriptConnection& other) const {
         return !(*this == other);
     }
+    private:
+    const RuntimeLoggingCategory m_logger;
```

Comment:
How to access and use ``qCCritical(m_logger) << "Critical Error: Triggering a connection "
                                    "inside the callback is not allowed.";`` inside the ScriptConnection.